dockerfiles/squid-proxy/latest/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM aosqe/squid-proxy
```

Comment:
We base the `squid-proxy` image on the `squid-proxy` image? Looks like an infinite additional of new layers to me.

Reply:
I think we need to original Dockerfile for `aosqe/squid-proxy`

Reply:
But I don't know where the `original` Dockerfile is and what it looks like. Do you know it?

Reply:
@openshift/team-qe anybody knows where the original Dockerfile is?

Reply:
@openshift-qe/openshift-qe-team ,  anybody knows where the original Dockerfile is?

Reply:
Hmm, clicking the @'ed name, it shows 404, seems the right one is openshift/team-qe

Reply:
> git blame features/step_definitions/helper_services.rb | grep " 150)"
b01384d9a7 (Chris Ryan            2016-09-28 12:57:36 -0500 150)     | app=squid-proxy |

This helps me find @cjryan committed https://github.com/openshift/cucushift/commit/b01384d9a7 and find https://github.com/openshift-qe/docker-squid/blob/master/Dockerfile
I'll re-build the image based on that and then update the new Dockerfile in this PR